raycicmd/step_filter.go

```diff
@@ -22,12 +22,29 @@ func (f *stepFilter) reject(step *stepNode) bool {
 }
 
 func (f *stepFilter) accept(step *stepNode) bool {
+	if f.selects != nil && f.tags != nil {
+		// in both key selection mode, hit when the step has both keys and tags
+		return f.acceptSelectHit(step) && f.acceptTagHit(step)
```

Comment:
I feel that we can make this function always return:

````
f.acceptSelectHit(step) && f.acceptTagHit(step)
````

and treat missing `selects` always as return `true`?

probably needs to change the `runAll` field in the `stepFilter` into `runAllTags` now, as it semantically only applies to tags selection.